hook_functions/tcp_seq_show.c

```diff
@@ -0,0 +1,15 @@
+asmlinkage int (*org_tcp4_seq_show) (struct seq_file *seq, void *v);
+
+// /proc/net/tcp show entry iterates over all tcp sockets and call tcp4_seq_show upon each one.
+// sk points either to 'SEQ_START_TOKEN' or to a socket, the special value is used at the beginning of the
+// iteration in order to print the title. 
```

Comment:
which title? what ends the begging of the iteration? Do you hide also listening sockets or only established ones?

Reply:
I hope the new documentation answers those questions

Reply:
indeed:)